src/test/java/com/wikia/webdriver/testcases/mcfootertests/EnLoggedInUserMixedContentFooterTests.java

```diff
@@ -0,0 +1,84 @@
+package com.wikia.webdriver.testcases.mcfootertests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mcfooter.MixedContentFooter;
+
+import org.testng.annotations.Test;
+
+@Test (groups = {"EnLoggedInUserMixedContentFooter"})
+@Execute(onWikia = "gameofthrones", asUser = User.USER)
+public class EnLoggedInUserMixedContentFooterTests extends NewTestTemplate{
+
+  @Test
+   public void mcFooterIsPresentOnENwiki() {
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
```

Comment:
@qaga .scrollToFooter() method is intended to scroll to global footer, not MCF. Depending on the screen resolution when test is executed (and jenkins has lower than you ;) and what exactly test is doing, test may fali. IMO we should write a method .scrollToMixContentFooter() and use it here and other tests instead of .scrollToFooter